src/config/dynamic-theme-fixes.config

```diff
@@ -2599,6 +2599,13 @@ img.listview__image, img.galleryview__image, .variant__header__dropdown__item__i
 
 ================================
 
+genius.com
+
+INVERT
+.texmath
```

Comment:
Example URL couldn't find this.

Reply:
Any song, really. Just scroll a bit down and see how the Apple Music bottom bar looks like.
https://genius.com/Luis-fonsi-despacito-lyrics